config-model/src/test/java/com/yahoo/vespa/model/application/validation/change/search/AttributeChangeValidatorTest.java

```diff
@@ -76,6 +93,15 @@ public void changing_fast_access_require_restart() throws Exception {
                                                   "Field 'f1' changed: remove attribute 'fast-access'"));
     }
 
+    // TODO Not testable until one dictionary type supports both cased/uncased
+    @Ignore
+    public void changing_uncased2cased_require_restart() throws Exception {
```

Comment:
Consider removing this until it is actually possible to specify.

Reply:
I prefer keeping it as a reminder, I think it will be possible before end of sprint.